server/prisma/migrations/20250630220811_lat_long_events/migration.sql

```diff
@@ -0,0 +1,6 @@
+-- AlterTable
+ALTER TABLE "Event" ADD COLUMN     "latitude" DECIMAL(65,30) NOT NULL DEFAULT 0.0,
+ADD COLUMN     "longitude" DECIMAL(65,30) NOT NULL DEFAULT 0.0;
```

Comment:
```suggestion
ALTER TABLE "Event" ADD COLUMN     "latitude" DECIMAL(65,30) NOT NULL,
ADD COLUMN     "longitude" DECIMAL(65,30) NOT NULL;
```

I would not add default values for non-null fields, they should be enforced in the business logic